### ISSUE TITLE
Developing a mardown-badges library

### ISSUE THREAD

**Issue by cybercoder-naj** [Ileriayo/markdown-badges#527]
## The issue 

Currently, this repository provides links to badge images that people can copy paste into their markdown files. However, from time to time, these badges break such as icons go missing without the developer knowing about it (for maybe months)

## Proposal

Host these images somewhere so that, in case of any breaking changes in the shields library, we can fix it and all users will have the updated version of the badge.

**Comment by kohasummons** [comment]
Absolutely! I've seen cases of several badge icons going down. I think most of the issue can be traced to the [SimpleIcons service](https://github.com/simple-icons/simple-icons) that Shields pull from. I read a `gh` issue a while back where Simple Icon had to take down an icon because the org/company that owns the Icon requested it. Notwithstanding, I'd love to hear your thoughts and maybe more details about the proposal.

**Comment by Jukelyn** [comment]
I disagree, if the icons are being taken off of SimpleIcons, it's almost always because of some brand guideline or trademark change. Those should be respected.
